client/signup/steps/rewind-were-backing/index.jsx

```diff
@@ -25,21 +26,27 @@ class RewindWereBacking extends Component {
 
 		// Connected props
 		siteSlug: PropTypes.string.isRequired,
+		dependencyStore: PropTypes.object,
 	};
 
 	stepContent = () => {
-		const { translate, siteSlug } = this.props;
+		const { translate, siteSlug, dependencyStore } = this.props;
 
 		return (
 			<Card className="rewind-were-backing__card rewind-switch__card rewind-switch__content">
 				<h3 className="rewind-were-backing__title rewind-switch__heading">
-					{ translate( "We're backing up your site!" ) }
+					{ translate( 'Your site is set up and ready!' ) }
 				</h3>
 				<img src="/calypso/images/illustrations/thankYou.svg" alt="" />
 				<p className="rewind-were-backing__description rewind-switch__description">
+					{ get( dependencyStore, 'rewindconfig', false ) &&
+						translate(
+							'Your site is backing up because your site is set up with Jetpack Premium plan at ' +
```

Comment:
This wording is pretty awkward, suggest:

"Your site is being backed up because it is set up with Jetpack Premium at no additional cost to you."

Reply:
(same above)

Reply:
Done 👍